### ISSUE TITLE
Cannot find environment variable to change basic_auth username

### ISSUE THREAD

**Issue by jerrywoo96** [kasmtech/workspaces-core-images#9]
Hi,

I'm spinning up the virtual desktops as individual containers, i see the environment variable to change the password, but not the variable to change the basic auth username login. What is the environment variable to change this? There was no documentation or docker-compose file that mentions this. I have only found the environment variable to change the basic auth password.

Thanks.

**Comment by jerrywoo96** [comment]
I've also read the issues tab, there was mention of this issue, but did not specify any solution.

**Comment by shkpk** [comment]
`kasm_user` is hard coded hence there is no env variable to change this. However you can search for `kasm_user` string in `vnc_startup.sh` file and change it to variable and then pass it while creating container.

**Comment by mmcclaskey** [comment]
That is correct, when using the full platform, the session username and password of KasmVNC are not relevant. You could use the [KasmVNC API](https://github.com/kasmtech/KasmVNC/wiki/API#add-user) to add a new user, assign ownership, and delete the kasm_user user. We are working to make the images more usable directly, without using our full platform and this is likely one thing we will address (custom username).
